docs/source-pytorch/common/trainer.rst

```diff
@@ -1525,6 +1525,39 @@ Whether to enable or disable the model summarization. Defaults to True.
 
     trainer = Trainer(enable_model_summary=True, callbacks=[ModelSummary(max_depth=-1)])
 
+
+eval_inference_mode
```

Comment:
I would suggest dropping the `eval_` prefix.

Reply:
Agree, inference describes all three stages in our Trainer: validate, test, predict.